### DISCUSSION TITLE
Emgu cv 3.1.xx Ip cam connection

### DISCUSSION THREAD

**Discussion by JCrovve** [emgucv/emgucv#479] [General]
Hey could any one give a Code example For a ip capture using emgu cv 3.1.x.x

I have a Pzt optics Hd box cam im trying to grab frames from but am having  zero luck with it.

**Comment by emgucv** [comment]
You can use this constructor:
https://www.emgu.com/wiki/files/4.5.1/document/html/219dd707-ca8f-374d-2ed4-f0e8dc612ace.htm

See Open CV documentation here:
https://docs.opencv.org/4.5.1/d8/dfe/classcv_1_1VideoCapture.html#ac4107fb146a762454a8a87715d9b7c96

The file name can be:
` URL of video stream (eg. protocol://host:port/script_name?script_params|auth)`

e.g. If your camera support RTSP, read the documentation to find the RTSP port & path. 
It could look something like the following:
`VideoCapture capture = new VideoCapture("rtsp://192.168.1.2:8080/out.h264");`
